### DISCUSSION TITLE
Controlling digits after decimal point

### DISCUSSION THREAD

**Discussion by monolithMktg** [cables-gl/cables_docs#350] [Answers & Questions] [answered]
I am looking for the equivalent op to do the toFixed() function on JS to limit some of the float numbers lengths. Thanks. :)

**Comment by pandrr** [comment, marked as answer]
https://cables.gl/op/Ops.String.RightPadNumber_v2

there are also leftpad and rightpad ops, round op controls the number of decimal places

**Reply by monolithMktg** [reply]
Brilliant!! Thank you very much Pandrr :D
